src/GovUk.Education.ExploreEducationStatistics.Common/Utils/Html/HtmlToTextConverter.cs

```diff
@@ -148,40 +170,58 @@ private void ParseTextNode(INode node)
                 return;
             }
 
-            // As there are potentially inline elements such as span/strong/em,
-            // we should try and add an extra space to avoid the text bunching up.
+            var prev = node.PreviousNonWhitespaceSibling();
+
+            if (prev is null)
+            {
+                if (node.Parent is IElement parentElement && parentElement.IsInlineType())
```

Comment:
Might be worth a few comments around the usage of "prev", "prevElement" and "parentElement" for the uninitialted if they come to this down the line.  I'm guessing here that we're saying for instance that if this is the first child nested within a parent element, then we need to be breaking out of the parent and seeing what the parent's previous sibling was in order to determine if whitespace is necessary or not.  Clear to me now after I've studied it a bit, but I had to think about it :slightly_smiling_face:

Reply:
Good point! I've updated the comments in this area of code and significantly bumped up the level of detail and context, if it looks good to you now?

Reply:
Yup, that's perfect, thanks!